### PR TITLE
Include IV and alter the default titles when parsing

### PR DESCRIPTION
**Pull request by roj1512**

### CODE REVIEW

File: bot.ts [hunk 1]
```diff
@@ -123,26 +123,31 @@ async function makeReplacements(
   text: string,
 ): Promise<[string, MessageEntity[], string]> {
   const searchQueries = new Array<string>();
+  text = `${ZWSP}${text}`;
```
Comment (KnorpelSenf) [inline bot.ts:126]: I don't think we can start messages with spaces

File: bot.ts [hunk 2]
```diff
@@ -123,26 +123,31 @@ async function makeReplacements(
...
   text.replace(replacementExp, (_, s, o) => {
     s = s.split("|");
     searchQueries.push(s[0]);
     return s;
   });
-  const urls = new Array<string>();
+  const hits = new Array<any>();
```
Comment (KnorpelSenf) [inline bot.ts:132]: Do you want to reuse the type definitions I added, rather than removing more and more?

**Review by KnorpelSenf — APPROVED** [review]
How did this get lost lol

**Comment by roj1512** [comment]
Questionable. Lmao.

**Pull request merged**